### ISSUE TITLE
Firefox placeholder opacity

### ISSUE THREAD

**Issue by thomaswelton** [necolas/normalize.css#277]
After scanning the issues I think this has already been discussed #157 as well as others.
But firefox is applying an opacity to the placeholder text color. While other browsers are not.

I'm using normalize via bootstrap. Today I received an issue on a project I'm working on that the text color for placeholders was different for firefox, which is why I'm looking into this now.

I'll submit a PR to reset the opacity. If it doesn't get merged then at least this issue can be found by others with a definitive answer about the opacity problem. This has been discussed in other issues but not explicitly.

**Comment by christopheschwyzer** [comment]
Some time has passed since this issue has been discussed - this still bothers me :)
It seems that browser vendors haven't made the effort to agree on a solution. I personally favorite FF's approach: _Inherit_ the color from the input, fade it out using _opacity_. 

Unfortunately, when trying to _normalize_ to this behavior IE will result in applying the opacity to the whole input field?!

```
::-webkit-input-placeholder {
   color: inherit;
   opacity: .54;
}
:-ms-input-placeholder {  
   color: inherit;  
   opacity: .54;
}
```

So the only way to have a similar appearance across browsers is like you mentioned in the bootstrap link. 

tl;dr
Nothing new. Oh and same discussion here: https://github.com/necolas/normalize.css/issues/157
